CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ option (PNET_OPTION_SNMP "" OFF)
 option (PNET_USE_ATOMICS "Enable use of atomic operations (stdatomic.h)" OFF)
 
 set(PNET_MAX_AR                 2
-  CACHE STRING "Number of connections. Must be > 0. 'Automated RT Tester' uses 2")
+  CACHE STRING "Number of connections. Must be > 0. 'Automated RT Tester' uses 2, but only 1 proper AR is supported.")
```

Comment:
Can we find another word than "proper". Maybe "connection"?

Reply:
Done